### PR TITLE
define single to HA actions

### PR DESCRIPTION
**Pull request by njuettner**
define all action to trigger HA upgrade

- create a cluster with a single master without any node pool (it should be fine)
- once it has been verified it runs switch to HA
- afterwards delete the crs and verify all things have been removed

## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by njuettner** [comment]
tested in ginger

File: pkg/action/cl006/ac007/executor.go [hunk 1]
```diff
@@ -1,9 +0,0 @@
-package ac007
```
Comment (njuettner) [inline pkg/action/cl006/ac007/executor.go:1]: I removed the ac007, this is not needed

**Comment by njuettner** [comment]
going with a ping :)

**Comment by paurosello** [comment]
I think it would make sense that we also check that the G8SControlPlane has been successfully modified via webhook, WDYT?

**Comment by njuettner** [comment]
I would like to see this in the integration tests for the admission controller tbh. I would expect also when we verify the replicas inside the cluster it should fail.

We can discuss this where we want to see those things and I agree we should check it, for now I would like to go with this approach. We can add it later if it should be defined in here.

**Comment by paurosello** [comment]
sure, all good

**Pull request merged**